src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/types.js

```diff
@@ -1,15 +1,15 @@
 // @flow
 export type RouteConfig = {|
-    attributeDefaults: AttributeMap,
+    attributeDefaults?: AttributeMap,
     name: string,
-    options: Object,
+    options?: Object,
     parent?: string,
     path: string,
-    rerenderAttributes: Array<string>,
-    view: string,
+    rerenderAttributes?: Array<string>,
+    type: string,
 |};
 
-export type Route = {|
+export interface Route {
```

Comment:
Do we really need that interface?

Reply:
We could remove it and use the `Route` class everywhere instead.